correctme/ui/gui.py

```diff
@@ -1,146 +1,78 @@
-from os.path import join, dirname
-
 from kivy.app import App
-from kivy.uix.label import Label
 from kivy.uix.gridlayout import GridLayout
-from kivy.uix.textinput import TextInput
-from kivy.uix.button import Button
-from kivy.lang import Builder
-
-Builder.load_file(join(dirname(__file__), r'SimpleKivy.kv'))
+from kivy.properties import ObjectProperty
+from kivy.uix.widget import Widget
```

Comment:
Do we need this `Widget` import? I can't find its usage anywhere.

Reply:
Yeah, missed it. Will fix it.